### DISCUSSION TITLE
MPC commands within a playlist - IE: turn randon/consume on/off, maybe others like pause/stop

### DISCUSSION THREAD

**Discussion by mikwei** [MusicPlayerDaemon/MPD#2049] [Ideas]
I have current playlist of tunes trying to organize and get new tunes.   So I add to end but if I could easily add a pre-item(not played) to turn on consume/off randon and at end turn-off consume and return to random play.  

Perhaps these could be *.mpc items vs an audio like mp3/flac... and add like an audio file but just executes and jumps to next.

Kinda nice if you have huge lists and you could click say Queen.mpc in playlist and play all songs until random.mpc encountered.   

Just food for thought, obviously some will want to turn on/off.  If no MPC commands then no diff.

**Comment by kingosticks** [comment]
Are you sure you can't already achieve what you want using the priority feature as explained in the docs at https://mpd.readthedocs.io/en/latest/protocol.html#queuing

**Comment by mikwei** [comment]
Queueing should work, and is definitely a feature I can make use of.

Issue I have is that none of the apps I am using seem to have a queue
option for me to test.  Oops I lied MPC seems to kinda support, but it's
not in the manual and only does single items(no range)...


𝓜𝓲𝓴𝓮 𝓦𝓮𝓲𝓼𝓼𝓮𝓷𝓫𝓸𝓻𝓷 ***@***.***>
